src/components/detailLikebutton/detailLikeButton.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import { HeartOutlined, HeartFilled } from '@ant-design/icons';
+import { useNavigate } from 'react-router-dom';
+import './detailLikeButton.scss';
+
+function HeartButton({ productId, isHeart, setIsHeart }) {
+	const navigate = useNavigate();
+
+	const activeLogin = () => {
+		fetch('http://localhost:8000/products/heart?productId=' + productId, {
+			headers: new Headers({ Authorization: sessionStorage.getItem('token') }),
+		})
+			.then(res => res.json())
+			.then(data => {
+				if (data.message === 'VALIDATE_ERROR') navigate('/users/login');
+			});
```

Comment:
```suggestion
		fetch(BASE_URL + '/products/heart?productId=' + productId, {
			headers: new Headers({ Authorization: sessionStorage.getItem('token') }),
		})
			.then(res => res.json())
			.then(data => {
				if (data.message === 'VALIDATE_ERROR') navigate('/users/login');
			});
```

https://github.com/wecode-bootcamp-korea/fullstack3-1st-wepleshop-frontend/blob/main/src/config.js